packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -413,15 +427,17 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
   const teamMemberHost = hosts.find((host) => host.user.email === input?.teamMemberEmail);
 
   // If the requested team member is a fixed host proceed as normal else get availability like the requested member is a fixed host
-  const usersWithCredentials =
-    !input.teamMemberEmail || !teamMemberHost || teamMemberHost.isFixed
+  const usersWithCredentials = [
+    ...(!input.teamMemberEmail || !teamMemberHost || teamMemberHost.isFixed
       ? hosts.map(({ isFixed, user }) => ({ isFixed, ...user }))
       : hosts.reduce((usersArray, host) => {
           if (host.isFixed || host.user.email === input.teamMemberEmail)
             usersArray.push({ ...host.user, isFixed: host.isFixed });
 
           return usersArray;
-        }, [] as (GetAvailabilityUser & { isFixed: boolean })[]);
+        }, [] as (GetAvailabilityUser & { isFixed: boolean })[])),
+    ...(attendeeCalUsers ? attendeeCalUsers.map((user) => ({ ...user, isFixed: false })) : []),
```

Comment:
> @Amit91848 Currently it's ready to review again, and I have a question regarding isFixed in the util.ts, I don't know what this data is for, I set the default for all attendees top false.


For users who have isFixed: true (collective event hosts and if user adds fixed hosts for round robin events) they **all have to available** at the given time slot. It is optional for non fixed users (round robin hosts where only one of hosts needs to be present)

So in this case we want attendee users to also be present, isFixed here should be true
```suggestion
    ...(attendeeCalUsers ? attendeeCalUsers.map((user) => ({ ...user, isFixed: true })) : []),
```